ndk/Cargo.toml

```diff
@@ -18,6 +18,7 @@ all = ["audio", "bitmap","media", "api-level-30"]
 audio = ["ffi/audio", "api-level-26"]
 bitmap = ["ffi/bitmap"]
 media = ["ffi/media"]
+midi = ["ffi/midi", "api-level-29", "media"]
```

Comment:
This wasn't dependent on `libmediandk.so` before; why is it now?

Reply:
This is because media_error.rs needs the "media" feature, would adding a separate "media_error" feature and making both "media" and "midi" be dependent on "media_error" okay?

Reply:
If I remember correctly I designed this to be extended (since it's not really a public feature):

https://github.com/rust-mobile/ndk/blob/e66b4fd71bc0fa1ae895a98506c51eb7cf588b68/ndk/src/media_error.rs#L5

Reply:
Building `ndk` only with "midi" and "api-level-29" makes error because `ndk::midi` depends on `ndk::media_error`. `ndk::media` is guarded with `#![cfg(feature = "media")]` while `ndk::midi` is not...

Reply:
Yes that works, I just didn't come up with that... 😢